tests/parsing/FilterTest.py

```diff
@@ -110,12 +110,12 @@ def test_filter_by_can_fix_null(self):
                 coala.main, 'coala', '-B', '--filter-by', 'can_fix')
             self.assertEqual(retval, 0)
             # 8 bears plus 1 line holding the closing colour escape sequence.
-            self.assertEqual(len(stdout.strip().splitlines()), 14)
+            self.assertEqual(len(stdout.strip().splitlines()), 15)
```

Comment:
Please add a separate PR to put this number in a variable in `tests.TestUtilities` (https://github.com/coala/coala/issues/4545), so we dont need to change these lines all the time.
Ping maintainers to get it merged ASAP, so you can rebase this PR.